lib/player.js

```diff
@@ -8259,8 +8259,9 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
     // Update the abr manager with newly filtered variants.
     const adaptationSet = this.currentAdaptationSetCriteria_.create(
         playableVariants);
-    this.abrManager_.setVariants(Array.from(adaptationSet.values()));
-    return true;
+    const ret =
+        this.abrManager_.setVariants(Array.from(adaptationSet.values()));
+    return typeof ret == 'boolean' ? ret : true;
```

Comment:
`updateAbrManagerVariants_()` returns `false` when error occurs, pls update method description and check does it influence anything.